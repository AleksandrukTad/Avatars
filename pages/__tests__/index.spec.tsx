import "jest-dom/extend-expect"
import "jest-styled-components"

// __tests__/fetch.js
import React from "react"
import {
  render,
  cleanup,
} from "@testing-library/react"
// this adds custom jest matchers from jest-dom

import Index from "../index"

// automatically unmount and cleanup DOM after the test is finished.
afterEach(cleanup)

test('', async () => {
  const { asFragment } = render( <Index />);
  // you can also get a `DocumentFragment`, which is useful if you want to compare nodes across renders
  expect(asFragment()).toMatchSnapshot();
})