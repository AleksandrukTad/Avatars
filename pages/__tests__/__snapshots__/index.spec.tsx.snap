// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<DocumentFragment>
  .c0 {
  font-weight: bold;
}

<div>
    Welcome to Next.js!
    <div
      class="c0"
    >
      Hello World
    </div>
  </div>
</DocumentFragment>
`;
